config/roles.yml

```diff
@@ -38,6 +40,8 @@ alerting_ack_alerts:
   reserved: true
   cluster_permissions:
     - 'cluster:admin/opendistro/alerting/alerts/*'
+    - 'cluster:admin/opendistro/alerting/workflow_alerts/*'
+    - 'cluster:admin/opendistro/alerting/chained_alerts/*'
```

Comment:
nit: Please add roles in alphabetical order

Reply:
Now we need to fix it since we have a task in the pipeline.
`Code Hygiene / Check permissions orders (pull_request)`

Reply:
Fixed this.